docs/packages/ratings/example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "page": "/components/cards",
+  "rating": 5,
+  "comment": "Yay!"
+}
```

Comment:
I think we want to include LIB_VERSION here. We can't rely on the date because older deploys exist and next/master deploys exist. It's also easier to filter instead of having to separately map dates to deployed versions.

And for debugging the netlify deploy ID.

Reply:
Good idea.